tests/drivers/can/api/boards/nucleo_n657x0_q_stm32n657xx_sb.conf

```diff
@@ -0,0 +1 @@
+CONFIG_TEST_USERSPACE=n
```

Comment:
Does this SoC not support userspace? Or is it just a matter of available MPU regions (if so, take a look at `CONFIG_TEST_USERSPACE_WITHOUT_HW_STACK_PROTECTION`)?

Either way, I think this would be better to set this from the board files instead of directly in each test for scalability. The CAN tests can't be the only ones affected by this limitation?

Reply:
The problem lies with the MPU. On N6, the bootROM loads the application from external Flash into the RAM where it is executed. I suppose that, for userspace to work, the region in the RAM where the code lies should be configured as the Flash usually is, but I haven't looked at that for the moment.

I'll move CONFIG_TEST_USERSPACE=n in the board files.

Reply:
Oh. Right. 😞